### PR TITLE
Mongoose models

### LINKED ISSUE #4: Crear entidades de base de datos
Crear entidades mongo necesarias para todo el proyecto.

AC:
- Crear los modelos en `api/src/models/`
- Todos los modelos de base de datos necesarios creados.

Ejemplo:
```javascript
// models/ingredient.js 
const mongoose = require('mongoose')
const { Schema } = mongoose

const IngredientSchema = new Schema({
    name: { type: String, required: true },
    type: { type: String, required: true },
    description: { type: String },
    image_url: { type: String },
    price: { type: Number, required: true },
    zindex: { type: Number, required: true }
})

module.exports = mongoose.model('Ingredient', IngredientSchema)
```

### PR DESCRIPTION
**Pull request by tr0b**
# Changes Include

## 8660a7c
- Added Initial Usuario Mongoose Model

## 15fe8e2
- Added Initial Rol Mongoose Model

## d05feab
- Added Initial Restaurante Mongoose Model

## f260972
- Added Initial Registro Mongoose Model

## c37a897
- Added Initial Proveedor Mongoose Model

## d2cd300
- Added Initial Producto Mongoose Model

## 225c489
- Added Initial Mesa Mongoose Model

## b1d9cee
- Added Initial Marca Mongoose Model

## e6eb371
- Added Initial Factura Mongoose Model

## 92154bc
- Added Initial Especialidad Mongoose Model

## 77c585a
- Added Initial Consecutivo Mongoose Model

## 820c397
- Added Initial Empleado Mongoose Model

## dca517a
- Added Initial Comestible Mongoose Model

## e810dbe
- Added Initial Cliente Mongoose Model

## cb16eb8
- Added Initial Bitacora Mongoose Model

## 4c60a5d
- Added Initial Bebida Mongoose Model

## 69ffc02
- Refactored Puesto Model import ObjectId call
- Refactored rol type ObjectId to include ObjectId directly from import
- Applied ESLint style to Puesto Model

## 8f0509f
- Applied ESLint to Pais Model

## 519f337
- Applied EsLint to Evento Model

## 9ed5ecc
- imported directly objectId from mongoose schema types
- refactored objectid type call in medida property

## 7379dd3
- Added Puesto model to models list for Mongoose
- Added Initial documentation for Puesto model (description)

## 7677994
- Added Evento Model to models list.
- Added initial documentation for Evento. (description and example)

## fba3482
- Added Buffet Schema to models list
- Added initial documentation for Buffet model (including description)

## 166f3ca
- Removed misleading documentation about Pais, from Medida model

## debcee0
- Created Initial Medida schema.
- Added initial documentation to medida schema, explaining its
  representation

## c15d7e1
- Added documentation to Pais Schema (description and properties)

## 61e239e
- Fixed ref nacionalidad to reflect Pais

### CODE REVIEW

File: api/src/models/bebida.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+/** @description Representacion de un Bebida de un Restaurante */
+const BebidaSchema = new Schema({
+  codigo: { type: String },
+  nombre: { type: String },
+  descripcion: { type: String },
+  ingredientes: { type: String },
+  marca: { type: ObjectId, ref: 'Marca' },
+  restaurante: { type: ObjectId, ref: 'Restaurante' },
+  nacionalidad: { type: ObjectId, ref: 'Medida' },
```
Comment (pieza) [inline api/src/models/bebida.js]: En nacionalidad creo que seria ref: 'Pais'

**Review by pieza — CHANGES_REQUESTED** [review]
Detalle menor

File: api/src/models/bebida.js (hunk 1 above)
Comment (tr0b) [reply api/src/models/bebida.js]: Changes implemented in commit ~> 61e239e

**Review by pieza — APPROVED** [review]
LGTM

**Pull request merged**